connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskTest.java

```diff
@@ -91,6 +94,11 @@
     // These are fixed to keep this code simpler. In this example we assume byte[] raw values
     // with mix of integer/string in Connect
     private static final String TOPIC = "test";
+    private static final Headers EMPTY_HEADERS = new RecordHeaders();
+    private static final Headers HEADERS = new RecordHeaders();
```

Comment:
Neither of these is really immutable, right? Wouldn't it be possible for a test method to modify those `Headers` instances? So would it be better to make these non-static and non-final -- or create new `RecordHeaders` instances whenever they are needed?

Reply:
Refactored to use `emptyHeaders()` + specific headers for the test